packages/frontend/src/routes/WalletWrapper.js

```diff
@@ -64,7 +67,7 @@ export function WalletWrapper({
             createFromImplicitSuccess={createFromImplicitSuccess}
             createCustomName={createCustomName}
             zeroBalanceAccountImportMethod={zeroBalanceAccountImportMethod}
-            fungibleTokensList={fungibleTokensList}
+            fungibleTokensList={blacklist.allowedTokens}
```

Comment:
I tried to remove black-listed tokens as high in hierachy as possible, to avoid multiple filtrations. 
If there is other places where token list will be used directly from the store - tokens will not be filtered there.
Please point if there is such places.

Reply:
This would also be good to add in the filter step of `components/send/components/views/SelectToken.js`